plugins/octopus-deploy/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@backstage/plugin-octopus-deploy",
   "version": "0.2.7",
-  "main": "src/index.ts",
-  "types": "src/index.ts",
+  "main": "dist/index.esm.js",
+  "types": "dist/index.d.ts",
```

Comment:
This breaks things, how come you made this change?

Reply:
oops, sorry, I was using that for local testing, thought I had reverted, my bad. Will fix this and ci now.

Reply:
Looks like all of the tests are passing now with that reverted + adding the api reports @Rugvip